src/model/comment-model.js

```diff
@@ -11,4 +11,5 @@ const getCommentData = (item) => ({
 export default class CommentsModel {
   comments = getCommentsListMock();
   getCommentsList = () => this.comments.map((value) => getCommentData(value));
```

Comment:
функция, создающая глубокую копию массива достойна быть самостоятельной

```suggestion
const deepArrayCopy = (array)=>array.map((item)=>({...item}));
//....
  getCommentsList = () => deepArrayCopy(this.comments);
```